src/xo-app/host/tab-patches.js

```diff
@@ -42,11 +46,11 @@ const MISSING_PATCH_COLUMNS = [
   },
   {
     name: _('patchAction'),
-    itemRenderer: (patch, installPatch) => (
+    itemRenderer: (patch, {installPatch, _installPatchWarning}) => (
       <ActionRowButton
         btnStyle='primary'
-        handler={installPatch}
-        handlerParam={patch}
+        handler={_installPatchWarning}
+        handlerParam={{patch, installPatch}}
```

Comment:
Avoid creating new values (arrays, objects or functions) in `handlerParam`, simply create an inline function in `handler` which inject the correct params.